src/mlpack/tests/test_catch_tools.hpp

```diff
@@ -71,25 +97,54 @@ inline void CheckFields(const FieldType& a,
     CheckMatrices(a(i), b(i));
 }
 
+
+template <typename CubeType, typename ElemType = typename CubeType::elem_type,
```

Comment:
```suggestion
// Simple wrapper class to prevent copies of Armadillo cubes.
template <typename CubeType, typename ElemType = typename CubeType::elem_type,
```